### ISSUE TITLE
Nightly failure: webUISharingAcceptShares/acceptShares.feature:16

### ISSUE THREAD

**Issue by saw-jan** [owncloud/web#7677]
Build: https://drone.owncloud.com/owncloud/web/28547/52/14
Test Scenario: `webUISharingAcceptShares/acceptShares.feature:16`

log:
```feature
  Scenario: reject a share that you received as user and as group member # features/webUISharingAcceptShares/acceptShares.feature:16
- Connecting to selenium on port 4444...

ℹ Connected to selenium on port 4444 (12908ms).
  Using: chrome (104.0.5112.79) on Linux platform.

    Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no" in the server
    And the administrator has set the default folder for received shares to "Shares" in the server
    And these users have been created with default attributes and without skeleton files in the server:
      │ username │
      │ Alice    │
      │ Brian    │
    And user "Brian" has logged in using the webUI
√ Element <input[autocomplete="kopano-account username"]> was visible after 600 milliseconds.
√ Element <input[autocomplete="kopano-account username"]> was not present after 185 milliseconds.
√ Element <#files-view> was visible after 673 milliseconds.
√ Element <//*[self::table[contains(@class, "files-table")] or self::div[contains(@class, "files-empty")] or self::div[contains(@class, "files-not-found")]]> was visible after 139 milliseconds.
    Given these groups have been created in the server:
      │ groupname │
      │ grp1      │
    And user "Alice" has created folder "/simple-folder" in the server
    And user "Brian" has been added to group "grp1" in the server
    And user "Alice" has shared folder "/simple-folder" with user "Brian" in the server
    And user "Alice" has shared folder "/simple-folder" with group "grp1" in the server
    And the user has browsed to the shared-with-me page
√ Element <//*[self::table[contains(@class, "files-table")] or self::div[contains(@class, "files-empty")] or self::div[contains(@class, "files-not-found")]]> was present after 446 milliseconds.
    When the user declines share "simple-folder" offered by user "Alice Hansen" using the webUI
√ Element <//span[contains(@class, "oc-resource-name") and (@data-test-resource-name='simple-folder' or @data-test-resource-path='/simple-folder')]/ancestor::tr[contains(@class, "oc-tbody-tr")]//td[contains(@class,"oc-table-data-cell-owner")]//span[@data-test-user-name="Alice Hansen"]/ancestor::tr[contains(@class, "oc-tbody-tr")]//button[contains(@class,"file-row-share")][normalize-space(.)="Decline"]> was visible after 38 milliseconds.
    And the user browses to the shared-with-me page in declined shares view
√ Element <//*[self::table[contains(@class, "files-table")] or self::div[contains(@class, "files-empty")] or self::div[contains(@class, "files-not-found")]]> was present after 535 milliseconds.
    Then folder "simple-folder" shared by "Alice Hansen" should be in "Declined" state on the webUI
    ✖ failed
      AssertionError [ERR_ASSERTION]: Expected resource 'simple-folder' shared by 'Alice Hansen' to be in state 'Declined' but isn't.
          + expected - actual

          -false
          +true

          at World.<anonymous> (/var/www/owncloud/web/tests/acceptance/stepDefinitions/sharingContext.js:801:19)
          at processTicksAndRejections (internal/process/task_queues.js:95:5)
    When the user browses to the files page
    - skipped
    Then folder "/Shares" should not be listed on the webUI
    - skipped
```

**Comment by SagarGi** [comment]
This tests seems to be failing when running multiple times 
```15 scenarios (6 failed, 9 passed)
225 steps (6 failed, 12 skipped, 207 passed)
3m38.307s (executing steps: 3m38.089s)
```
I will look in further into it.

**Comment by SagarGi** [comment]
Video screenshot of the failure: 

https://user-images.githubusercontent.com/46086950/191933426-b8d43917-6b7b-4fd4-8123-a27e046e2a4d.mp4

When declined is clicked then there is a page reload step that reloads the page and the share is still in the pending state which fails the tests. This does not happens always but sometimes. And also when user declines the share the to view the decline state it doesnot require the reload. I will debug that in CI with the reload step remove.

**Comment by saw-jan** [comment]
Failed again in: https://drone.owncloud.com/owncloud/web/28588/52/14

**Comment by SagarGi** [comment]
> Failed again in: https://drone.owncloud.com/owncloud/web/28588/52/14

i will debug it in CI and make a PR for it.

**Comment by SagarGi** [comment]
Locally this tests seems to be failing as : 
```28 scenarios (7 failed, 21 passed)
420 steps (7 failed, 14 skipped, 399 passed)
7m00.732s (executing steps: 7m00.368s)
```

but also in debugging in CI it seems to be passed : 
https://drone.owncloud.com/owncloud/web/28606/10/19

But With removing the reload step it is consistent with even locally running the tests as: 
```
28 scenarios (28 passed)
392 steps (392 passed)
6m54.712s (executing steps: 6m54.344s)
```

Removing the reload step will make it consistent and not flaky.

**Comment by saw-jan** [comment]
Appeared again: https://drone.owncloud.com/owncloud/web/28680/52/14
@SagarGi your fix PR (https://github.com/owncloud/web/pull/7688) has got approval. Could you merge it if it is ready?

**Comment by phil-davis** [comment]
I merged PR #7688

**Comment by saw-jan** [comment]
Fix PR has been merged.
Closing here!
